AlphaWallet/Settings/Types/Constants.swift

```diff
@@ -58,6 +58,7 @@ public struct Constants {
     public static let burnAddressString = "0x000000000000000000000000000000000000dEaD"
     static let nullAddress = AlphaWallet.Address(uncheckedAgainstNullAddress: "0x0000000000000000000000000000000000000000")!
     static let nativeCryptoAddressInDatabase = nullAddress
+    static let ensContractOnMainnet = nullAddress
```

Comment:
@hboon can u please check this one, it was mised in prev PR.

Reply:
Got it in. Thanks